main/mainworker.cpp

```diff
@@ -2055,7 +2058,7 @@ void MainWorker::ProcessRXMessage(const CDomoticzHardwareBase *pHardware, const
 
 	const_cast<CDomoticzHardwareBase *>(pHardware)->SetHeartbeatReceived();
 
-	char szDate[100]; szDate[0] = 0;
+	char szDate[100];
```

Comment:
Recent change?

I had this branch uploaded four days ago. The master branch in my fork is the same age. Wanted to run some checks before pushing them as some users reported that the sunrise/sunset based schedules did fire at (more or less) the correct time.